src/05_lists.py

```diff
@@ -8,22 +8,43 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+
+# x.append(int(4))
+
+# or
+
+x.insert(3,int(4))
+
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+y=[8,9,10]
+x.extend(y)
+
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE
+
+x.pop(4)
```

Comment:
this pops the item at index 4 out of the list? very cool! 
you could also use ``` x.remove(8) ``` here.  great job!